Dockerfile

```diff
@@ -14,14 +14,14 @@ ENV TERM xterm
 
 ENV TERM xterm
 
+#The solr config file needs to be added after installation or it fails.
+ADD docker_files/solr /etc/default/solr.docker
```

Comment:
Hi Andy! Thank you for your contribution! The .docker append is to evict solr warn that the file already exists or something like that?